atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -137,18 +136,8 @@ static void runWithWaitingForSchemas(
         waitForSchemaVersions(config, client, "after " + unsafeSchemaChangeDescription);
     }
 
-    static boolean uniqueSchemaWithQuorumAgreementAndOtherNodesUnreachable(
-            CassandraKeyValueServiceConfig config,
-            Map<String, List<String>> versions) {
-        List<String> reachableSchemas = getDistinctReachableSchemas(versions);
-        if (reachableSchemas.size() > 1) {
-            return false;
-        }
-
-        int numberOfServers = config.servers().numberOfThriftHosts();
-        int numberOfVisibleNodes = getNumberOfReachableNodes(versions);
-
-        return numberOfVisibleNodes >= ((numberOfServers / 2) + 1);
+    static boolean reachablePartOfClusterHasConsistentSchemaVersions(Map<String, List<String>> versions) {
+        return getDistinctReachableSchemas(versions).size() == 1;
```

Comment:
it probably doesn't matter what we return in that case? expectation being your schema mutation will fail because there is no online node for it to succeed against.